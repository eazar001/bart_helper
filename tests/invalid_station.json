{
  "version": "1.0",
  "session": {
    "new": true,
    "sessionId": "amzn1.echo-api.session.cb96e737-7d5a-4c10-89bf-d7698ab8f888",
    "application": {
      "applicationId": "amzn1.ask.skill.2909ff08-8fcd-404c-9d25-c51f32ff2355"
    },
    "user": {
      "userId": "amzn1.ask.account.AGCZQMVG6R45GB4J6ARK26GZP5KWHKNBOXKC42W7AUSI2MHTXCS4H7TCFCBFNDNL6QNMQ3GKHQCMYIX27ZM4AY7TLEEC47ZM7MEVDOIYYQYH5B5B4AOILYBKSFDOLMWGFTIMOQVOTPCSPFZX35POCP2NQGVWDUWEESFC3RGZ6ZUD2B5UUDUDPKHZJQWC67M62FQLSYSPTFVJDLY"
    }
  },
  "context": {
    "System": {
      "application": {
        "applicationId": "amzn1.ask.skill.2909ff08-8fcd-404c-9d25-c51f32ff2355"
      },
      "user": {
        "userId": "amzn1.ask.account.AGCZQMVG6R45GB4J6ARK26GZP5KWHKNBOXKC42W7AUSI2MHTXCS4H7TCFCBFNDNL6QNMQ3GKHQCMYIX27ZM4AY7TLEEC47ZM7MEVDOIYYQYH5B5B4AOILYBKSFDOLMWGFTIMOQVOTPCSPFZX35POCP2NQGVWDUWEESFC3RGZ6ZUD2B5UUDUDPKHZJQWC67M62FQLSYSPTFVJDLY"
      },
      "device": {
        "deviceId": "amzn1.ask.device.AHVWONRCVS7NFVWIPULLBGNVMX2EO65IGMQHVARCF67NQERL56FYE7SQ3K6GR4OVXSLTY2AN7RWHNS2H34BBIAOBW2AW3IQ6SDCRGFBAUVRPYH6NEZW2NJZFKE5HXTRWLJPHN5J5NC6QORDOWX2CK3SM3MXQ",
        "supportedInterfaces": {}
      },
      "apiEndpoint": "https://api.amazonalexa.com",
      "apiAccessToken": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IjEifQ.eyJhdWQiOiJodHRwczovL2FwaS5hbWF6b25hbGV4YS5jb20iLCJpc3MiOiJBbGV4YVNraWxsS2l0Iiwic3ViIjoiYW16bjEuYXNrLnNraWxsLjI5MDlmZjA4LThmY2QtNDA0Yy05ZDI1LWM1MWYzMmZmMjM1NSIsImV4cCI6MTU1NzYyMjE5OSwiaWF0IjoxNTU3NjIxODk5LCJuYmYiOjE1NTc2MjE4OTksInByaXZhdGVDbGFpbXMiOnsiY29uc2VudFRva2VuIjpudWxsLCJkZXZpY2VJZCI6ImFtem4xLmFzay5kZXZpY2UuQUhWV09OUkNWUzdORlZXSVBVTExCR05WTVgyRU82NUlHTVFIVkFSQ0Y2N05RRVJMNTZGWUU3U1EzSzZHUjRPVlhTTFRZMkFON1JXSE5TMkgzNEJCSUFPQlcyQVczSVE2U0RDUkdGQkFVVlJQWUg2TkVaVzJOSlpGS0U1SFhUUldMSlBITjVKNU5DNlFPUkRPV1gyQ0szU00zTVhRIiwidXNlcklkIjoiYW16bjEuYXNrLmFjY291bnQuQUdDWlFNVkc2UjQ1R0I0SjZBUksyNkdaUDVLV0hLTkJPWEtDNDJXN0FVU0kyTUhUWENTNEg3VENGQ0JGTkROTDZRTk1RM0dLSFFDTVlJWDI3Wk00QVk3VExFRUM0N1pNN01FVkRPSVlZUVlINUI1QjRBT0lMWUJLU0ZET0xNV0dGVElNT1FWT1RQQ1NQRlpYMzVQT0NQMk5RR1ZXRFVXRUVTRkMzUkdaNlpVRDJCNVVVRFVEUEtIWkpRV0M2N002MkZRTFNZU1BURlZKRExZIn19.KuERLVfCjIaGC-LyyZ9673fiOC_6k-iB1GTXIUrGTUsPZkHBCsKBDRqHSENdHJZLjfR9UK02zrS0mVrb5EU0W2Xqq7b75ZRbi_ns1BuHseHovWue_RaGE6K8VND-BvX25JkiPUvnYQwJPDpZEt6r9Dp0i2a2KF-dydGA7N-a9t6ShWXVYgiJEzM6RiwjiHwB2mcB4ZR-sphgJ2JGyKaX_xGEChhNxRkDTVVRMRyFj9U3nRcBaRDdqrJ0ZC12G8jBQ8eyISqeO49bYs8myboJ_-gVX0GFRJGw7WHHESFq8t6WQzXfDgSCXAsUMFXpOSdmXfzapkZ0xB_FivrwRpPZ5A"
    }
  },
  "request": {
    "type": "IntentRequest",
    "requestId": "amzn1.echo-api.request.4f7aed1f-35f5-4fa7-874c-0eddb12c092f",
    "timestamp": "2019-05-12T00:44:59Z",
    "locale": "en-US",
    "intent": {
      "name": "fare",
      "confirmationStatus": "NONE",
      "slots": {
        "origin": {
          "name": "origin",
          "value": "A",
          "confirmationStatus": "NONE"
        },
        "dest": {
          "name": "dest",
          "value": "B",
          "confirmationStatus": "NONE"
        }
      }
    }
  }
}