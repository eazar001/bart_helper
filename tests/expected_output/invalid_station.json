{
  "version": "1.0",
  "response": {
    "outputSpeech": {
      "type": "PlainText",
      "text": "Sorry I thought I heard you say \"a\", but that is not a valid station.\nPlease try again."
    },
    "card": {
      "type": "Simple",
      "title": "Invalid station",
      "content": "Sorry I thought I heard you say \"a\", but that is not a valid station.\nPlease try again."
    },
    "shouldEndSession": true
  }
}